table/sstable/iterator.go

```diff
@@ -297,7 +357,7 @@ func (itr *Iterator) seekFrom(key []byte) {
 	itr.err = nil
 	itr.reset()
 
-	idx := itr.seekBlock(key)
+	idx := itr.seekBlockWithPlr(key)
```

Comment:
Once can change this single line to `itr.seekBlock(key)` to turn off PLR (bourbon) feature